instrumentation/rocketmq-client-4.8/README.md

```diff
@@ -0,0 +1,6 @@
+# Settings for the Apache RocketMQ client instrumentation
+
+| System property | Type | Default | Description |
+|---|---|---|---|
+| `otel.instrumentation.rocketmq-client.experimental-span-attributes` | Boolean | `false` | Enable the capture of span attributes (experimental). |
+| `otel.instrumentation.rocketmq-client.propagation` | Boolean | `true` | Enable propagation. |
```

Comment:
You could probably use similar/same description as in the kafka-clients propagation property.